client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.ts

```diff
@@ -234,7 +234,7 @@ export class ParticipantDetailViewComponent extends BaseMeetingComponent {
         // TODO: Open groups subscription
         this.groups = this.groupRepo.getViewModelListWithoutSystemGroupsObservable();
 
-        this.disableExpandControl = this.user.vote_delegations_from();
+        this.disableExpandControl = this.user?.hasVoteRightFromOthers ? this.user.vote_delegations_from() : [];
```

Comment:
Wouldn't it suffice to rewrite `vote_delegations_from` so it returns [] if there are no delegations?

Reply:
Okay, I looked at the error message again, that looks like either the user or the meeting_user has not yet loaded at that point. Probably need to ensure that happens first or maybe add a cheeky little '?' after the offending variable, before calling `vote_delegations_from`. Your choice.